src/acl.c

```diff
@@ -3073,6 +3079,7 @@ void aclCommand(client *c) {
             if (count < 0) count = 0;
         }
 
+        trimACLLogEntriesToMaxLen();
```

Comment:
I suppose I don't understand why we need to trim here.

Reply:
Discussed we would like to stick with the existing behavior of trimming while new entries get created.